index.html

```diff
@@ -3109,7 +3124,8 @@ <h2>Definition of Roles</h2>
 			<rdef>figure</rdef>
 			<div class="role-description">
 				<p>A perceivable <rref>section</rref> of content that typically contains a <a>graphical document</a>, images, code snippets, or example text. The parts of a <code>figure</code> MAY be user-navigable.</p>
-				<p>Authors SHOULD provide a reference to the <code>figure</code> from the main text, but the <code>figure</code> need not be displayed at the same location as the referencing element. Authors MAY reference text serving as a caption using <pref>aria-describedby</pref>. Authors MAY provide a label using <pref>aria-label</pref> or MAY reference text serving as a label using <pref>aria-labelledby</pref>.</p>
+				<p>Authors SHOULD provide a reference to the <code>figure</code> from the main text, but the <code>figure</code> need not be displayed at the same location as the referencing element. Authors MAY reference text serving as a caption using <pref>aria-describedby</pref>.</p>
```

Comment:
this is confusing now:
>Authors MAY reference text serving as a caption using `aria-describedby`

as the `caption` is meant to provide the accessible name of the `figure`, we shouldn't also say it should be referenced by `aria-describedby`.